### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by DeamonFire**
### HTML-CSS-Popup

[Demo](https://deamonfire.github.io/HTML-CSS-Popup/) |
[Code base](https://github.com/DeamonFire/HTML-CSS-Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by DeamonFire** [comment]
Hi, yes i checked my code according to all the parameters specified in the recommendation

File: submissions/DeamonFire/HTML-CSS-Popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
+
```
Comment (A-Ostrovnyy) [inline submissions/DeamonFire/HTML-CSS-Popup/css/style.css:1]: Avoid too many empty(unused) lines in your code.

File: submissions/DeamonFire/HTML-CSS-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+                    <li class="header__menu_block">
+                        <div class="header__menu_block_hidden popup-menu">
+                            <input type="checkbox" id="menu_1" class="popup-menu__input">
+                            <label for="menu_1" class="popup-menu__label" aria-label="Advanced menu"></label>
+                            <div class="popup">
+
+                                    <ul class="popup_hidden wrapper">
+                                        <li class="wrapper__block">
+                                            <a href="" class="wrapper__block_link">
```
Comment (A-Ostrovnyy) [inline submissions/DeamonFire/HTML-CSS-Popup/index.html:30]: Use `#` as the value of an empty `href` attribute. Some kind of convention.

File: submissions/DeamonFire/HTML-CSS-Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+                                        </li>
+                                    
+                                    </ul>           
+                            </div>
+                        </div>
+                    </li>
+
+                    <li class="header__menu_block">
+                        <a href="" class="header__menu_block_link">
+                            <img src="popup-icons/bell.png" alt="bell" class="header__img">
```
Comment (A-Ostrovnyy) [inline submissions/DeamonFire/HTML-CSS-Popup/index.html:218]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/DeamonFire/HTML-CSS-Popup/index.html [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+                                                    <img src="popup-icons/calendar.png" class="wrapper__block_img">
+                                                    <div class="wrapper__block_title">Calendar</div>
+                                                </div>
+                                            </a>
+                                        </li>
+                            
+                                        <li class="wrapper__block">
+                                            <a href="" class="wrapper__block_link">
+                                                <div class="wrapper__block_link_item">
+                                                    <img src="popup-icons/mail.png" class="wrapper__block_img">
```
Comment (A-Ostrovnyy) [inline submissions/DeamonFire/HTML-CSS-Popup/index.html:41]: The `alt` is the required attribute for `img` element, if it's empty just add it with empty value
```HTML
<img alt="" >
```

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Sorry for the late response.

**Comment by A-Ostrovnyy** [comment]
Also, don't forget about mobile version 
image

File: submissions/DeamonFire/HTML-CSS-Popup/css/style.css (hunk 1 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/css/style.css]: Hi, i removed extra empty lines

File: submissions/DeamonFire/HTML-CSS-Popup/index.html (hunk 2 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html]: Done

File: submissions/DeamonFire/HTML-CSS-Popup/index.html (hunk 3 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html]: i fixed it

File: submissions/DeamonFire/HTML-CSS-Popup/index.html (hunk 4 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html]: Fixed that as well

**Comment by DeamonFire** [comment]
Hi, I fixed the comment

File: submissions/DeamonFire/HTML-CSS-Popup/index.html [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+                        <a href="#" class="header__menu_block_link">
+                            <img src="popup-icons/bell.png" alt="Sign up for updates" class="header__img">
+                        </a>
+                    </li>
+
+                    <li class="header__menu_block">
+                        <a href="#" class="header__menu_block_link">
+                            <img src="popup-icons/account.png" alt="Go to your account page" class="header__img">
+                        </a>
+                    </li>
```
Comment (A-Ostrovnyy) [inline submissions/DeamonFire/HTML-CSS-Popup/index.html:227]: What about these alts?

File: submissions/DeamonFire/HTML-CSS-Popup/index.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="css/style.css">
+        <title>Hooli-style Popup</title>
+    </head>
+    <body>
+        <header class="header">
+            <h1 class="header__h1">Hooli-style Popup</h1>
+            <nav class="header__nav">
+                <ul class="header__menu">
```
Comment (A-Ostrovnyy) [inline submissions/DeamonFire/HTML-CSS-Popup/index.html:13]: Avoid shifting sibling elements on hover or focus. [Watch video](https://share.vidyard.com/watch/8oDnqwEyGidcU2frYpoxJ9?)
Also, check the hit-box of your elements: when the user hovers `li` element you display the border but the cursor is the default, and you change it when the users hover the link inside `li`. All changes on hover must be at the same time.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

File: submissions/DeamonFire/HTML-CSS-Popup/index.html (hunk 5 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html:227]: Can you tell me what this error is, I can't figure it out on my own (

File: submissions/DeamonFire/HTML-CSS-Popup/index.html (hunk 6 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html:13]: Solved this problem =)

File: submissions/DeamonFire/HTML-CSS-Popup/index.html (hunk 5 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html]: I rewrote them more succinctly. Need to remove alt tags from these icons? But without the voiceover element, they will be invisible to the screen reader

Comment (A-Ostrovnyy) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html]: Look, for images inside the link in the popup you set an empty value for `alt` attribute but for images in the link, you set the text value for the `alt` attribute. Try to use a single approach for all images.)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

File: submissions/DeamonFire/HTML-CSS-Popup/index.html (hunk 5 above)
Comment (DeamonFire) [reply submissions/DeamonFire/HTML-CSS-Popup/index.html]: Ні, I fixed it
I hope my solution to this comment is correct =)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**